corehq/apps/smsbillables/migrations/0005_smsbillable_twilio_gateway_fee.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0004_bootstrap_smsgh_rates'),
+    ]
+
+    operations = [
```

Comment:
@snopoke and @gcapalbo are the experts, but my understanding is that it likely will be a bit of a problem (though not sure how much we can do about it)